bugsnag/client.py

```diff
@@ -117,12 +120,24 @@ def deliver(self, notification):  # type: (Notification) -> None
         if not self.should_deliver(notification):
             return
 
+        initial_severity = notification.severity
+        initial_default_severity = notification.default_severity
+        initial_unhandled = notification.unhandled
+        initial_severity_reason = notification.severity_reason
+
         def send_payload():
             if notification.api_key is None:
                 bugsnag.logger.warning(
                     "No API key configured, couldn't notify")
                 return
 
+            notification.default_severity = initial_default_severity
```

Comment:
I dont think we need to reset any changes to these variables in here. If people change them then they are in expert mode and thats fine.